stories/step-config.js

```diff
@@ -0,0 +1,128 @@
+const props = Object.freeze({
```

Comment:
maybe maintain this as the "default" step configs so this can be mostly backward compatible with the current snoke

Reply:
Sure, also, I'm not sure what the `geo_cohort` relationship/prop is, can you verify that part?

Reply:
ah sure.

play on the current builder UI to confirm what I state here though. `geo_cohort` depends on `report` and AOI enrichment steps (the current builder UI auto matically checks those when you check `Geo Cohort`)

![geo_cohort](https://user-images.githubusercontent.com/2837532/131581347-5ab32c44-6b45-45d5-ac66-acd256548ec0.png)

Reply:
So something like this?
![image](https://user-images.githubusercontent.com/44912260/131720868-d9b439bb-6538-4fa6-90ee-2ae5e2b23296.png)

Reply:
also in a case like this, should it connect to both reports?

Reply:
yep!